src/native-http-backend.ts

```diff
@@ -167,6 +148,45 @@ export class NativeHttpBackend implements HttpBackend {
         });
     }
 
+    private buildRequestOptions(req: HttpRequest<any>, requestMethod, headers) {
+        let serializerType = this.detectDataSerializerType(req);
+        const requestOptions: any = {
```

Comment:
Could you please specify a new type
`type SendRerquestOptions = Parameters<typeof HTTP.prototype.sendRequest>[1]` at the beginnign of file and use this type for `requestOptions` instead of `any` ?

Reply:
Good idea - done 👍